content/en/Getting started/glossary.md

```diff
@@ -121,9 +121,7 @@ An _Organization Owner_ is the administrator for a customer organization within
 
 - An _Organization Owner_ may also be a [_Pentest Team Member_](#pentest-team-member).
 
-- If allowed by their {{% ptaas-tier %}}, they can also manage
-[multi-factor authentication (MFA)](#multi-factor-authentication) as well as
-[SAML](#security-assertion-markup-language) settings for the users in their organization.
+- Manage [multi-factor authentication (MFA)](#multi-factor-authentication) as well as [SAML](#security-assertion-markup-language) settings for the users in their organization.
```

Comment:
And now I realize that I was not consistent here.... What do you think? We could:

- Change this to 2FA
- Explain that in our app, users can set up MFA through "Two-factor authentication"

Reply:
@mjang I would add one sentence to Glossary > Multi-factor Authentication:

We support two-factor authentication with codes from a mobile authenticator app.

Reply:
Sounds good to me. I might even make it shorter:

```suggestion
- Manage [multi-factor authentication (MFA)](#multi-factor-authentication) as well as [SAML](#security-assertion-markup-language) settings for the users in their organization. We support two-factor authentication with a mobile authenticator app."
```